src/dbspro/cli/run.py

```diff
@@ -0,0 +1,88 @@
+"""
+Run DPS-Pro pipeline
+"""
+
+import logging
+import os
+from pathlib import Path
+import sys
+import pkg_resources
+from snakemake import snakemake
+
+from dbspro.utils import available_cpu_count
+
+logger = logging.getLogger(__name__)
+
+accepted_file = 'reads.fastq.gz'
+accepted_file_ext = '.fastq.gz'
+
+
+def main(args):
+
+    cwd = Path(os.getcwd())
+
+    # Check if path to output directory is absolute or make it so.
+    if not os.path.isabs(args.directory):
+        args.directory = cwd / args.directory
+
+    # Check if directory exists. If not make one.
+    if not os.path.isdir(args.directory):
+        os.mkdir(args.directory)
+        logging.info(f'Output directory {args.directory} created.')
+
+    # If no targets --> run whole pipeline i.e. use final file names.
+    if not args.targets:
+        args.targets = ['umi-counts.txt', 'umi-density-plot.png', 'read-density-plot.png']
```

Comment:
This is probably better to have as default in the argparser.
```
default='umi-counts.txt umi-density-plot.png read-density-plot.png'
```

Reply:
That is true, changing in new commit.